dashboard/modules/version.py

```diff
@@ -1,7 +1,10 @@
 from dataclasses import dataclass
 
 # Version 0 -> 1: Added log streaming and changed behavior of job logs cli.
-CURRENT_VERSION = "1"
+# Version 1 -> 2: - Renamed job_id to submission_id.
+#                 - Changed list_jobs sdk/cli/api to return a list
+#                   instead of a dictionary.
+CURRENT_VERSION = "2"
```

Comment:
is this intended? or an artifact of stacked PRs? please make this a separate PR either way.

Reply:
Intended but yes, an orthogonal change. Will separate.

Reply:
Thanks. LGTM to merge after this is resolved.